src/common/schemas/specifications/payment.json

```diff
@@ -22,11 +22,11 @@
       "$ref": "hash256"
     },
     "allowPartialPayment": {
-      "description": "A boolean that, if set to true, indicates that this payment should go through even if the whole amount cannot be delivered because of a lack of liquidity or funds in the source account account",
+      "description": "If true, this payment should proceed even if the whole amount cannot be delivered. This can occur due to a lack of liquidity or a lack of funds in the source account.",
```

Comment:
"This can occur" makes it sound almost like the `allowPartialPayment` flag could get turned on when this situation occurs. Suggested rephrase:

> If true, this payment should proceed even if the whole amount cannot be delivered due to a lack of liquidity or a lack of funds in the source account.